### ISSUE TITLE
Containerized

### ISSUE THREAD

**Issue by Dale4** [graphql-dotnet/graphql-dotnet#691]
Is it possible to add the GraphQL nuget package into an asp.net core project that can be containerized by docker on a windows 2016 server?

**Comment by joemcbride** [comment]
Yes. My experience with Docker is minimal, though my understanding is that you create your docker container which copies your compiled asp.net core project to a base image, which would include the dlls from this project.

https://docs.docker.com/engine/examples/dotnetcore/
